# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/GC.h
# Commit: 098569654f17
# Full Hash: 098569654f17fe958cfa7ca7077e4d770044a62e
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-28 21:22:11
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r 6e5a935e576a -r 098569654f17 js/src/gc/GC.h
--- a/js/src/gc/GC.h	Fri Oct 28 09:47:14 2022 +0000
+++ b/js/src/gc/GC.h	Fri Oct 28 09:47:14 2022 +0000
@@ -49,7 +49,6 @@
   _("unusedChunks", JSGC_UNUSED_CHUNKS, false)                             \
   _("totalChunks", JSGC_TOTAL_CHUNKS, false)                               \
   _("sliceTimeBudgetMS", JSGC_SLICE_TIME_BUDGET_MS, true)                  \
-  _("markStackLimit", JSGC_MARK_STACK_LIMIT, true)                         \
   _("highFrequencyTimeLimit", JSGC_HIGH_FREQUENCY_TIME_LIMIT, true)        \
   _("smallHeapSizeMax", JSGC_SMALL_HEAP_SIZE_MAX, true)                    \
   _("largeHeapSizeMin", JSGC_LARGE_HEAP_SIZE_MIN, true)                    \